containers/scripts/docker-compose-data-services.yml

```diff
@@ -1,5 +1,15 @@
 ---
 services:
+  mira:
+    container_name: mira
+    image: ghcr.io/darpa-askem/mira:latest
+    networks:
+      - terarium
+    environment:
+      MIRA_NEO4J_URL: 'bolt://neo4j:7687'
```

Comment:
Does this mean we won't be able to strip out neo4j from or tech stack? Or is this like an embeded neo4j that runs inside mira container?

Reply:
The latter, it's is own neo4j. So when we are done with the DKG, it's just removing one single image from the stack.